README.md

```diff
@@ -77,7 +117,7 @@ Copy a geojson polygon
 #TODO Maybe the whole projects directory should be in a volume
 docker cp /path/to/my/random_polygon.geojson transition_transition-www_1:/app/projects/demo_transition/imports/polygon.geojson
 ### TODO ADD  downloadosmtask....
```

Comment:
We could take the opportunity to replace this TODO line with the actual download task.